apps/strapi-cms/src/components/parts/markdown.json

```
@@ -0,0 +1,17 @@
+{
+  "collectionName": "components_parts_markdowns",
+  "info": {
+    "displayName": "Markdown",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "dirName": {
+      "type": "string",
+      "required": true
+    },
+    "pathToFile": {
+      "type": "text"
```

A Markdown part without a path is useless:
```suggestion
      "type": "text",
      "required": true
```